style2.css

```diff
@@ -400,4 +452,459 @@
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5200px;
+}
+
+.enter {
+  height: 114px;
+  width: 327px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d0d9d4;
+  margin-left: 15px;
+  position: absolute;
+  top: 5270px;
+}
+
+.name:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.name-last:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.enter:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+}
+
+.touch-btn {
+  background-color: #36b37c;
+  color: #ebf0ee;
+  font-size: 17px;
+  border-radius: 5px;
+  height: 48px;
+  width: 129px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 12px;
+  padding-right: 12px;
+  position: absolute;
+  top: 5480px;
+  left: 80px;
+}
+
+.touch-btn:hover {
+  text-decoration: underline rgb(6, 14, 243);
+  border-color: transparent;
+  box-shadow: 0 0 0 1px rgb(6, 14, 243);
+  cursor: pointer;
+}
+
+.icon-row2 {
+  display: flex;
+  position: absolute;
+  top: 5580px;
+  left: 78px;
+}
+
+.email-address {
+  width: 375px;
+  height: 189px;
+}
+
+.email {
+  color: #42526e;
+  font-size: 20px;
+  line-height: 32px;
+  position: absolute;
+  left: 70px;
+  top: 5620px;
+}
+
+.email:hover {
+  text-decoration: underline rgb(6, 14, 243);
+}
+
+.top-row li {
+  display: inline-block;
+}
+
+.bottom-row li {
+  display: inline-block;
+}
+
+.center-row li {
+  display: inline-block;
+  margin-bottom: 20px;
+}
+
+.contactpage {
+  display: flex;
+  flex-direction: column;
+}
+
+#user_name {
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+}
+
+#text-box {
+  width: 327px;
+  height: 114px;
+}
+
+.textline1 {
+  margin-left: 22px;
+  font-size: 38.5px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline2 {
+  margin-left: 10px;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline3 {
+  margin-left: 10px;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline4 {
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline5 {
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline6 {
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.textline7 {
+  margin-left: 50px;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+input[type='text'] {
+  font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+input[type='email'] {
+  font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+textarea {
+  font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#full-name {
+  border-width: 1px;
+  color: #6b778c;
+  margin-left: 10px;
+}
+
+#email {
+  border-width: 1px;
+  color: #d0d9d4;
```

Comment:
- [ ] [REQUIRED] Please change your color to correct one `#091E42;`

![image](https://user-images.githubusercontent.com/63915024/186476535-889428ca-028f-442d-9e80-025dda8d5fbe.png)

Reply:
ok changed